package_build.sh

```diff
@@ -0,0 +1,38 @@
+# This is the script overall responsible for building the pip package for
+# multiple versions of TensorFlow.
+
+set -e
+set -x
+
+if [[ -z ${1} ]]; then
+  echo "No output directory provided"
+  exit 1
+fi
+OUT=${1}
+
+if [[ -z ${2} ]]; then
+  echo "No temporary directory provided, creating new"
+  TMP=$(mktemp -d -t tfbig-XXXXXXX)
+else
+  TMP=${2}
+fi
+
+# make sure directories exist
+mkdir -p ${OUT}
+mkdir -p ${TMP}
+
+# manually needed metadata files
+./package_build_metadata.sh ${TMP}
+
+# performs builds
+pip install -U tensorflow==1.13.1 && ./package_build_tagged.sh ${TMP}
```

Comment:
You can use an array and a for loop to make this programmable :)

Reply:
note to self:
- https://unix.stackexchange.com/questions/140502/sh-how-to-use-array-in-sh-script-in-order-to-print-all-values-in-array
- https://stackabuse.com/array-loops-in-bash/